tutorials/klusterai-api/llm-as-a-judge.ipynb

```diff
@@ -23,26 +23,24 @@
    "id": "355a788f-a618-42a2-98c1-3d0e68ff766c",
    "metadata": {},
    "source": [
-    "In our previous <a href= \"https://github.com/kluster-ai/klusterai-cookbook/blob/main/examples/model-comparison.ipynb\" target=\"_blank\">notebook</a>, we explored the idea of selecting the best model to perform a classification task. We did that by calculating the accuracy of each model based on a ground truth label. In real-life applications, though, the ground truth is not always available, and to create one, we might depend on human annotation, which is time-consuming and costly. \n",
+    "How can we test a model's accuracy when the ground truth is unavailable? One approach could be to test the predictions made by the base model against a larger model, which, comparatively, should do better.\n",
+    "\n",
+    "This tutorial uses a base model (`Llama-3.1-8B-Instruct-Turbo`) to classify a dataset based on a description. Next, we will use a larger model (`klusterai/Meta-Llama-3.3-70B-Instruct-Turbo`) as a judge, tasked to determine whether the base model's predicitions are correct. Since the dataset also contains the ground truth, the notebook also assesses how well the judge model performed.\n",
```

Comment:
```suggestion
    "This tutorial uses a base model (`Llama-3.1-8B-Instruct-Turbo`) to classify a dataset based on a description. Next, we will use a larger model (`klusterai/Meta-Llama-3.3-70B-Instruct-Turbo`) as a judge, tasked to determine whether the base model's predictions are correct. Since the dataset also contains the ground truth, the notebook also assesses how well the judge model performed.\n",
```